### PR TITLE
Added ability to pull all reports for a given account

### PR DESCRIPTION
**Pull request by yosemsweet**

### CODE REVIEW

**Comment by yosemsweet** [comment]
Hey Sharla, do you mind reviewing this change to the kissr package. I can run you through it all. Documentation is light but things seem to work

**Comment by sharlagelfand** [comment]
In kissr.r, you say the package has two functions: to read and list. It should be to read and read.

**Comment by yosemsweet** [comment]
Need to update examples and documentation for KissReports

**Comment by yosemsweet** [comment]
Need to make read an S3 generic (http://adv-r.had.co.nz/S3.html)

File: R/kiss-reports.R [hunk 1]
```diff
@@ -4,10 +4,53 @@
...
+  }
+
+  # Get list of reports
+  links  <- jsonlite::fromJSON(httr::content(response, "text"))$links
+  statusLink <- links[links$name == "Status", "href"]
+
+  reportsLink <- with(result, links[links$name == "First", "href"])
+  print("Getting all reports")
+  results <- as.data.frame(loadPages(reports, reportsLink), stringsAsFactors = FALSE)
+  urlExtractor <- function(df) { df[3,3] }
```
Comment (sharlagelfand) [inline R/kiss-reports.R]: You should be explicit here about what grabbing the (3,3) entry is doing -- will this always be the entry you want? Why?

**Pull request closed without merging**